randovania/data/json_data/prime1.txt

```diff
@@ -2366,6 +2570,26 @@ Asset id: 2263559682
 > Dock to Life Grove Tunnel; Heals? False
   > Life Grove Tunnel/Dock to Life Grove
       Trivial
+  > Life Grove/Pickup (X-Ray Visor)
+      Trivial
+
+> Pickup (X-Ray Visor); Heals? False
+  > Life Grove/Behind PB wall
+      X-Ray Visor and Can lay power bomb
+
+> Pickup (Artifact of Chozo); Heals? False
+  > Life Grove/Behind PB wall
+      Trivial
+
+> Behind PB wall; Heals? False
+  > Life Grove/Dock to Life Grove Tunnel
+      Power Beam and X-Ray Visor and Space Jump Boots
+  > Life Grove/Pickup (X-Ray Visor)
+      X-Ray Visor and Can lay power bomb
+  > Life Grove/Pickup (Artifact of Chozo)
+      All of the following:
+          Can Boost
+          Can lay bomb or Can lay power bomb
 
 ----------------
 Transport to Phazon Mines East
```

Comment:
Add another node to this room called "Elevator to Phazon Mines - Transport to Tallon Overworld South" with the requirement Scan Visor.